utils/constants.py

```diff
@@ -11,13 +11,21 @@
 else:
 	raise Exception('This system is not supported.')
 
+##### DIRECTORIES
+
 DATA_DIR = SRC_DIR / 'data'
+
 TRENDS_DIR = DATA_DIR / 'trends'
 TRIMMED_DIR = DATA_DIR / 'trimmed'
 IMAGES_DIR = DATA_DIR / 'images'
-LOG_DIR = SRC_DIR / 'logs'
 SHIRTS_DIR = DATA_DIR / 'shirts'
-STATIC_DIR = SRC_DIR / 'static'
-SHIRT_BG = STATIC_DIR / 'background.jpg'
 SHOPIFY_JSON = DATA_DIR / 'json'
 RESPONSE_JSON = DATA_DIR / 'response'
+
+LOG_DIR = SRC_DIR / 'logs'
+STATIC_DIR = SRC_DIR / 'static'
+
+##### FILEPATHS
+
+SHIRT_BG = STATIC_DIR / 'background.jpg'
+ENV_PATH = SRC_DIR / '.env'
```

Comment:
newline here ;)

Reply:
Want me to add an extra line you're saying?